src/app/components/core/control-dispatcher/control-dispatcher.module.ts

```diff
@@ -58,11 +68,25 @@ const routes: Routes = [
     path: 'task/:name',
     component: ControlScheduleTaskDetailsComponent
   },
+  {
+    path: 'pipelines',
+    component: ControlPipelinesComponent,
+    canActivate: [AuthTypeGuard]
+  },
+  {
+    path: 'pipeline/add',
+    component: AddControlPipelineComponent,
+    canActivate: [AuthTypeGuard]
+  },
+  {
+    path: 'pipeline/:name',
```

Comment:
sidebar menu item got unselected on redirecting pipeline details page.
the correct route path now is. `pipeline/:name/details`,  like https://github.com/fledge-iot/fledge-gui/pull/286/files#diff-0b66eeed40b32de8b4f1286877f232e6b749220c47e1529bf89d0e115ccf5f7bR59